activestorage/test/service/disk_service_test.rb

```diff
@@ -3,16 +3,16 @@
 require "service/shared_service_tests"
 
 class ActiveStorage::Service::DiskServiceTest < ActiveSupport::TestCase
-  tmp_config = { tmp: { service: "Disk", root: File.join(Dir.tmpdir, "active_storage") } }
-  SERVICE = ActiveStorage::Service.configure(:tmp, tmp_config)
+  TMP_CONFIG = { tmp: { service: "Disk", root: File.join(Dir.tmpdir, "active_storage") } }
+  SERVICE = ActiveStorage::Service.configure(:tmp, TMP_CONFIG)
 
   include ActiveStorage::Service::SharedServiceTests
 
-  test "name" do
+  def test_name
```

Comment:
changed the style here

1) to match how tests in most of Rails are written
2) because `test_can_change_the_root` needs to be a method so it can use `ensure`

Reply:
@gmcgibbon I can change it back for all the other tests if you like, but I do need it for at least one.

Reply:
I suppose I could write a wrapper method actually... this feels much neater though.

Reply:
`ensure` has been allowed in blocks since Ruby 2.5. Since HEAD's minimum version is 2.7, this should _just work_ AFAIK.

Reply:
🤯

Reply:
that's so cool